src/js/content_scripts/input-mapper-background-utils.js

```diff
@@ -91,4 +91,41 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             chrome.runtime.sendMessage(actionData);
         }
     };
+
+    /**
+     *
+     * Sends message to the background script to open a new window.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Boolean} background - Whether the new window should open in background.
+     * @param {Integer} oldInputValue - The previous value of the gamepad input.
+     * @param {String} homepageURL - The URL for the new window.
+     *
+     */
+    gamepad.inputMapperUtils.background.openNewWindow = function (that, value, background, oldInputValue, homepageURL) {
```

Comment:
You now have a lot of functions like this where you screen for `value < oldInputValue` and then construct an action payload.  It seems like it's time to replace all six functions that call `chrome.runtime.sendMessage` with invokers that are backed by a common function.

This would also leave more of the key wiring declared as options, which I prefer to having so much of it "in code".